packages/js/client/src/__tests__/core/embedded-package.spec.ts

```diff
@@ -0,0 +1,173 @@
+import fs from "fs";
```

Comment:
are these on purpose? i guess they should only live in `core-client` and they are repeated?

Reply:
They are, they test embedded wrappers when added through the `PolywrapClient` which uses our default config with the config builder